### ISSUE TITLE
[ feature ] WebUI Mobile Support

### ISSUE THREAD

**Issue by LukeLalor** [eidolon-ai/eidolon#881]
right now the webui is bad on mobile. We need a cycle to make this beautiful 🤩.

Community tag since this comes from joseph.

**Comment by flynntsang** [comment]
It occurred to me that we might want to assemble an AWS AMI that we can stand up for Ravi to show off on his mobile phone. We might want to consider bumping this to P1. @LukeLalor @parmi02 @ravieidolon thoughts?

**Comment by LukeLalor** [comment]
I agree with this, in particular getting the github assistant working.

then ravi could ask it "are the devs working on right now" as a somewhat tongue in cheek mini demo. It would also be a good way to show the different multi-agent system since he could pop open the dev tools and show what is running.

If we want to prioritize that story, bumping this priority up would make sense, but there is other work to do there as well.
- [ ] adding auth to github assistant so only eidolon team members can use it
- [ ] deploying github assistant to cloud
- [ ] ci-cd to auto-deploy github assistant to cloud

**Comment by flynntsang** [comment]
Let's keep the priority as-is and work towards it 🙂
